servers.ini

```diff
@@ -0,0 +1,91 @@
+# individual servers
+
+[webserver2]
+109.123.121.248
+
+[webserver3]
+77.92.77.3
+
+[webserver5]
+77.92.77.20
+
+[webserver6]
+77.92.77.148
+
+[webserver8]
+37.123.113.46
+
+[webserver9]
+37.123.117.143
+
+[webserver10]
+83.170.85.222
+
+[webserver11]
+83.170.85.227
+
+[webserver12]
+176.56.237.212
+
+[webserver13]
+37.123.118.6
```

Comment:
...turns out the fix didn't work (well, if you don't specify the `all` list, things are bad).

As such, have commented out the IP addresses in https://github.com/IATI/deployment/pull/6